app/src/main/java/org/oppia/android/app/options/LanguageRadioButtonListener.kt

```diff
@@ -0,0 +1,5 @@
+package org.oppia.android.app.options
+
+interface LanguageRadioButtonListener {
```

Comment:
Please add documentation to this class & the method to explain what the listener is, when it can be expected to be called, what parts of the view/fragment/activity hierarchy should implement it (e.g. fragment vs. activity), and what the 'selected language' value represents.

Reply:
Done